fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ComposeTabDrawerRobot.kt

```diff
@@ -136,7 +136,9 @@ class ComposeTabDrawerRobot(private val composeTestRule: HomeActivityComposeTest
         }
     }
 
+    @OptIn(ExperimentalTestApi::class)
     fun verifyNormalTabsList() {
+        composeTestRule.waitUntilDoesNotExist(hasTestTag("tabstray.tabList.normal.empty"), waitingTime)
```

Comment:
Unless there is a "not Displayed" option here, LGTM.

Reply:
No, unfortunately there isn't.